app/src/main/java/com/github/sdp/tarjetakuna/utils/TemporaryCards.kt

```diff
@@ -0,0 +1,58 @@
+package com.github.sdp.tarjetakuna.utils
+
+import com.github.sdp.tarjetakuna.model.*
+
+/**
+ * Represents temporary cards that are generated with a loop
+ */
+object TemporaryCards {

```

Comment:
I invite you to comment on this code so as not to forget to delete it later, as it is a special situation.